packages/airnode-validator/src/config/config.test.ts

```diff
@@ -69,3 +69,71 @@ describe('nodeSettingsSchema', () => {
     );
   });
 });
+
+it('fails if ois.apiSpecifications.security.<securitySchemeName> is defined in ois.apiSpecifications.components.<securitySchemeName> but apiCredentials.securitySchemeValue is empty string', () => {
```

Comment:
`fails if ois.apiSpecifications.security.<securitySchemeName> is defined in ois.apiSpecifications.components.<securitySchemeName> but missing value in apiCredentials`

I don't think we should enforce non empty string here.